test/CMakeLists.txt

```diff
@@ -163,10 +163,28 @@ endif()
 
 foreach(SDK ${SWIFT_SDKS})
   foreach(ARCH ${SWIFT_SDK_${SDK}_ARCHITECTURES})
+    # macCatalyst needs to run two sets of tests: one with the normal macosx target triple
+    # and one with the the macCatalyst ios-macabi triple.  The build_flavors list will
+    # have have only the "default" flavor for all SDKs and architectures except
+    # OSX when macCatalyst support is enabled.
+    set(build_flavors "default")
+    if(SWIFT_ENABLE_MACCATALYST AND "${SDK}" STREQUAL "OSX")
+      list(APPEND build_flavors "ios-like" )
+    endif()
+
+    foreach(BUILD_FLAVOR ${build_flavors})
```

Comment:
Now that we're upstreaming this foreach loop, I think we can indent its body!

Reply:
Yes, but there are reasons we can't do it in this PR. I'll fix this in a follow-on PR.